pkg/apis/controller/common/v1alpha3/common_types.go

```diff
@@ -51,6 +51,8 @@ type ObjectiveSpec struct {
 	// This can be empty if we only care about the objective metric.
 	// Note: If we adopt a push instead of pull mechanism, this can be omitted completely.
 	AdditionalMetricNames []string `json:"additionalMetricNames,omitempty"`
+	// This field is allowed to missing, experiment defaulter (webhook) will fill it.
+	MetricStrategies map[string]MetricStrategy `json:"metricStrategies,omitempty"`
```

Comment:
After this change, I can see that you send Metrics to Suggestion according to MetricStrategies. Is it right way?
Maybe some Suggestions need min, max and latest Observation? Do we need to be consistent between manager: https://github.com/kubeflow/katib/blob/master/pkg/apis/manager/v1alpha3/api.proto#L177 and Trial API?
/cc @johnugeorge @gaocegege

Reply:
If we modify definition of observation in `api.proto`, we need to apply corresponding modifications on all suggestion services.

Reply:
Yes, I understand it. You think we don't need it in suggestions?

Reply:
@andreyvelich Yes. For suggestion service,  as far as I know, the reward of training program (F(x)) is the only necessary value.

Reply:
I think it is a huge change to API. Can we place this change in the next release? Maybe v1beta1.

Reply:
Agree with @gaocegege. Let's keep it for next release.